src/jetson/main.py

```diff
@@ -59,6 +63,20 @@ def __init__(self, detector:str, detection_threshold=0.7, cuda=True, set_default
             self.net.min_suppression_threshold = 0.3
             self.transformer = BaseTransform(128, None)
 
+        elif ('.pth' in detector and 'mobile' in detector):
+            from models.Retinaface.retinaface import RetinaFace, load_model
+
+            self.net = RetinaFace(cfg=cfg, phase = 'test')
+            self.net = load_model(self.net, detector, True)
+            self.model_name = 'retinaface'
+            self.image_shape = (480, 640)  #(H, W)
+            self.resize = 1  # Change this value by the factor which image_shape is changed
```

Comment:
this comment is really confusing. try and be more concise and describe what it is.